lms/static/sass/elements/_banners.scss

```diff
@@ -44,3 +44,50 @@ $full-width-banner-margin: 20px;
         right: $full-width-banner-margin;
     }
 }
+
+.page-banner {
+    max-width: $lms-max-width;
+    margin: 0 auto;
+
+    .user-messages {
+        padding-top: $baseline;
+    }
+
+    .alert {
+        margin-bottom: $baseline !important;
```

Comment:
Would putting this inside the user.messages remove the need for the !important or is this an override bootstrap issue?